src/pdn/include/pdn/PdnGen.hh

```diff
@@ -57,7 +57,9 @@ using odb::dbRegion;
 
 using utl::Logger;
 
+#ifndef SWIG
```

Comment:
I don't where `std::regex` is used in the header, I think it might be left over from when global connect was here. You can probably just remove it along with the header import

Reply:
will do.